### PR TITLE
Streamupload

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
- Install `youtube-dl` and `ffmpeg` tools needed for the streaming upload commands

### CODE REVIEW

File: run-system-tests/action.yml [hunk 1]
```diff
@@ -118,6 +118,18 @@ runs:
       with:
         go-version: 1.16
 
+    - name: "Download youtube-dl"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        sudo curl -L https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl
```
Comment (stewartie4) [inline run-system-tests/action.yml:124]: Pipe the output to a build log and only show on error like we do with the CLIs?
echo "Installing youtube-dl"
sudo curl -L https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl > build.log 2>&1 || { cat build.log && exit 1; }

File: run-system-tests/action.yml [hunk 2]
```diff
@@ -118,6 +118,18 @@ runs:
...
+        sudo chmod a+rx /usr/local/bin/youtube-dl
+
+    - name: "Donwload ffmpeg"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        sudo apt update
+        sudo apt install -y ffmpeg
```
Comment (stewartie4) [inline run-system-tests/action.yml:131]: same with this? helps keep the actions console output readable

File: run-system-tests/action.yml (hunk 1 above)
Comment (Kishan-Dhakan) [reply run-system-tests/action.yml]: Ah, neat!

File: run-system-tests/action.yml [hunk 3]
```diff
@@ -177,6 +200,8 @@ runs:
         exit_code=0
         echo "$SMART_CONTRACT_OWNER_WALLET_JSON" >> ./config/sc_owner_wallet.json
         echo '{"client_id": "591d6d0d5642bdbc924756ca2647b4b59adce0acd02a487ecc4d1bd4669293a8","client_key": "ac8798bcadd5ce5ea14f4745850eec61aa080e6fedc1d769af95c42e0079ba03ee9710e9dc966f7c5eb9cd6ef9184af51869d5a345b9520e507aa55ba22bf902","keys": [{"public_key": "ac8798bcadd5ce5ea14f4745850eec61aa080e6fedc1d769af95c42e0079ba03ee9710e9dc966f7c5eb9cd6ef9184af51869d5a345b9520e507aa55ba22bf902","private_key": "fbdc0bcc12168588e2def826b7ee9d8b4c6f1fbacf22ee5064b7db49482b8f0d"}],"mnemonics": "economy day fan flower between rebuild valid bid catch bargain vivid hybrid room permit check manage mean twelve damage summer close churn boat either","version": "1.0","date_created": "2021-12-04T17:03:37Z"}' >> ./config/blobber_owner_wallet.json
+        ~/go/bin/gotestsum --jsonfile stream-test-output.json --hide-summary=output --format testname ${{ env.SHOULD_RETRY_FAILURES }} --raw-command -- ./STREAM_TEST_RUNNER_COMMAND.sh || exit_code=$?
```
Comment (stewartie4) [inline run-system-tests/action.yml:203]: remember to revert this separation

**Review by stewartie4 — APPROVED** [review]

**Pull request merged**